services/connected_account.yaml

```diff
@@ -52,25 +37,22 @@ Parameters:
     Description: |
       Unique ExternalId for Customer Organization; for cross-account Role Access and
       associating this template with a Customer Organization
-  ReactorAccountId:
-    Type: String
-    Default: '061190967865'
-    Description: |
-      CloudZero AWS AccountID; for cross-account Role Access
-  ReactorId:
-    Type: String
-    Default: 'f7a07d02-d509-4e8d-9fb9-69c7985a6bd8'
-    Description: |
-      CloudZero Reactor ID; Used to Uniquely ID the CloudZero System Ingesting your Data
-  ReactorSQSQueueUrl:
-    Type: String
-    Default: 'null'
-    Description: |
-      CloudZero Reactor Queue Url; receives callbacks from this template
+
+
+Mappings:
+  CallbackConfiguration:
+    dev:
+      ReactorAccountId: '998146006915'
+      ReactorId: '67ad3e83-1ec5-4f4f-b505-c27e2758b990'
+      ReactorCallbackUrl: 'null'
+    prod:
+      ReactorAccountId: '061190967865'
+      ReactorId: 'f7a07d02-d509-4e8d-9fb9-69c7985a6bd8'
+      ReactorCallbackUrl: 'https://api.cloudzero.com/accounts/v1/link'
```

Comment:
This one concerns me, as we can't cover all of our namespaces with this approach.  Can we convert this to a custom resource at some point?  Then we could resolve the reactor ID via runtime config instead.

Reply:
I'll add a TODO

Reply:
Remember, these resources get deployed in our customers' accounts, where namespaces don't really apply.

Reply:
right, what I means was that each of those reactor IDs maps to one of our namespaces.

Reply:
Right right ... that makes sense.